.env.example

```
@@ -162,3 +162,51 @@ AWARENESS_DELIVERY_POLL_MS=2000
 # directly, so there are no AaaS-specific Neo4j vars.
 # Portal -> API base URL
 PUBLIC_AWARENESS_API_URL="http://localhost:4100"
+
+# W3DS -> OIDC bridge (services/w3ds-oidc-bridge)
+# Lets GitW3 accept W3DS login through Forgejo's native OAuth2 source.
+# Public base URL of the bridge. This is the OIDC `issuer`, and Forgejo compares
+# it byte for byte, so a trailing slash is a silent total failure - the service
+# strips one if present. Must be https:// outside local development: goth never
+# verifies the ID token signature, so TLS is what makes the token trustworthy.
+W3DS_OIDC_PUBLIC_URL="http://localhost:4200"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)